thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template-properties-builder/alert-template-sectioned-properties/alert-template-sectioned-properties.component.tsx

```diff
@@ -32,107 +34,125 @@ const StyledGrid = withStyles((theme) => ({
-                                                }
-                                            />
-                                        )
-                                    )}
-                                    {Object.keys(subStepMap)
-                                        .slice(1)
-                                        .filter(
-                                            (cs) => subStepMap[cs].length > 0
-                                        )
-                                        .map((currentSubStep) => (
-                                            <Grid
-                                                item
-                                                key={`required-${step}-${currentSubStep}`}
-                                                xs={12}
+                    return (
+                        <Grid item key={`existing-${step}`} xs={12}>
+                            <Box paddingBottom={2}>
+                                <Typography variant="h5">
+                                    {extendablePropertyStepNameMap[step]}
+                                </Typography>
+                                {subStepMap?.["DIRECT"]?.map(
+                                    (directSubStep, directSubStepIdx) => (
+                                        <AlertTemplateFormField
+                                            defaultValue={
+                                                directSubStep.metadata
+                                                    .defaultValue
+                                            }
+                                            item={directSubStep}
+                                            key={directSubStep.key}
+                                            tabIndex={directSubStepIdx + 1}
+                                            tooltipText={
+                                                directSubStep.metadata
+                                                    .description
+                                            }
+                                            onChange={(selected) =>
+                                                handlePropertyValueChange(
+                                                    directSubStep.key,
+                                                    selected
+                                                )
+                                            }
+                                        />
+                                    )
+                                )}
+                                {Object.keys(subStepMap)
+                                    .slice(1)
+                                    .filter((cs) => subStepMap[cs].length > 0)
+                                    .map((currentSubStep) => (
+                                        <Grid
+                                            item
+                                            key={`required-${step}-${currentSubStep}`}
```

Comment:
Please ensure that this is always unique.

Reply:
They are.